.travis.yml

```diff
@@ -35,5 +35,5 @@ script:
     - flake8 tests
     - flake8 examples
     - python tests/test_peninsula.py --grid 20 10
-    - py.test -v -s tests/
+    - py.test tests/
```

Comment:
Why would we want less verbosivity here? I liked being able to see which tests worked and which didn't. Is there a reason for this?